locales/en/LC_MESSAGES/volto.po

```diff
@@ -1441,17 +1441,17 @@ msgstr "Curriculum vitae"
 #: components/ItaliaTheme/View/Commons/TrasparenzaFields
 # defaultMessage: Risultati indagini di customer satisfaction
 msgid "customer_satisfaction"
-msgstr ""
+msgstr "Results of customer satisfaction surveys"
 
 #: components/ItaliaTheme/View/Commons/PageHeader/PageHeaderEventDates
 # defaultMessage: dal {dateStart} fino a conclusione
 msgid "dal {dateStart} fino a conclusione"
-msgstr ""
+msgstr "from {dateStart} until the end"
 
 #: components/ItaliaTheme/View/IncaricoView/IncaricoView
 # defaultMessage: Data conclusione incarico
 msgid "data_conclusione"
-msgstr ""
+msgstr "Date of completion of assignment"
```

Comment:
"of" non si usa quasi mai ed è un po' macchinoso
"Assignment completion date" molto più leggibile :)